src/openMVG/robust_estimation/guided_matching.hpp

```diff
@@ -41,7 +41,7 @@ void GuidedMatching(
   //  the smallest distance (smaller than the provided Threshold)
 
   #pragma omp parallel for
-  for (int i = 0; i < xLeft.cols(); ++i) {
+  for (size_t i = 0; i < xLeft.cols(); ++i) {
```

Comment:
I believe int is required here for visual studio compiler

Reply:
Okay, I am not familar with visual studio compiler. and i will change them to int.